strawberryfields/ops.py

```diff
@@ -87,28 +87,15 @@
   i.e., we may symbolically use the measurement result before it exists::
 
     with prog.context as (alice, bob):
-        MeasureFock()| alice
-        Dgate(alice) | bob
+        MeasureFock()    | alice
+        Dgate(alice.par) | bob
```

Comment:
This is a pretty big UI change (requiring `alice.par` rather than `alice` for measurement values) - is it possible to retain the previous behaviour? It would also require changes across the tutorials and blackbird

Reply:
I don't see it as a backbreaker. It's 3 extra lines and still pretty clear what's happening. Bigger question is the second one: what tutorials will we have to update?

Reply:
Making `Dgate(alice)` work would be easy (just have `Operation.__init__` convert the RegRef into a MeasuredParameter). The hard case is something like `Dgate(2 * alice.par**2)`. Here we cannot drop `.par` because `alice` alone is a RegRef instance and does not implement arithmetic operations. I think it's clearer if we always require the `.par`.